backend/app/services/diagnosis_agent.py

```
@@ -33,205 +33,529 @@ class DiagnosisState(TypedDict):
+  {weather_summary}
+- 조언: {weather_advice}
+
+## 🤖 AI 총평 및 방제 전략
+
+- {normalized_bullets[0]}
+- {normalized_bullets[1]}
+- {normalized_bullets[2]}
+
+## ⚠️ 공지: 제공된 정보는 공공데이터에 기반한 참고용입니다. 농약 사용 전 반드시 제품 라벨의 규정을 확인하십시오."""
+
+async def fetch_weather(state: DiagnosisState) -> dict:
```

Ruff (0.15.10)

[warning] Too many branches (56 > 12)

(PLR0912)